htdocs/ticket/class/ticket.class.php

```diff
@@ -500,8 +500,13 @@ public function create($user, $notrigger = 0)
 				if ($this->add_contact($this->fk_user_assign, 'SUPPORTTEC', 'internal') < 0) {
 					$error++;
 				}
-			}
+				$ret = $this->assignUser($user, $this->fk_user_assign);
```

Comment:
What happen if fk_user_assign is not set or empty (case by default) ?

Reply:
Just go deeper, I can simplify the code, the issue is that trigger TICKET_ASSIGNED is not checked in function create. Will change and come back to you